helpers.json

```diff
@@ -222,6 +222,14 @@
           "name": "h_SupersDoubleDamageMotherBrain",
           "requires": [ "never" ],
           "devNote": "In Vanilla, Supers do 300 damage to Mother Brain. Randomizers may want to double this to help prevent Charge being required so frequently."
+        },
+        {
+          "name": "h_BypassMotherBrainRoom",
```

Comment:
Is Morphless a reality 🪄 

This helper is meant to skip the zebs+map rando blockade+be in position to hit MB.
There was mention of a teleport to the left of MB and you could think the wall would be open since it is a door when entering from the left, but room state won't have it there I don't think.  As a way to travel through the room without fighting anything.
And I want to say Colosseum lets you teleport to the right door but I don't know what room state looks like.  And I find right side teleports harder to imagine.

Reply:
Yeah the intent is to use this helper in Map Rando to add requirements for the objectives to be logically complete. Getting to the MB in this way is fine, but we wouldn't want it to result in objectives not being completable. Being able to skip the objectives may be too powerful, so we might eventually also want to enforce them in some other way (e.g., not being able to hurt MB until they are complete).

The left door being already blown open is a thing that Map Rando adds to avoid getting instantly softlocked, and it only does it when entering through that door. So in Map Rando, teleporting to the left side of the room from the right side (e.g. from Moat or Red Brin Firefleas) doesn't seem to be useful, as you would just be stuck, unless the glass is already broken, but in that case you would have had to have gone through the room normally earlier.